front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -90,6 +95,21 @@ export type AssistantBuilderDataSourceConfiguration = {
   isSelectAll: boolean;
 };
 
+// DustAppRun Action
+
+const DUST_APP_MODES = ["GENERIC", "SELECTED"] as const;
+type DustAppMode = (typeof DUST_APP_MODES)[number];
+const DUST_APP_MODE_TO_LABEL: Record<DataSourceMode, string> = {
+  GENERIC: "None (Generic model)",
```

Comment:
I think `(Generic model)` is strange. Just `None` is probably fine ?

Reply:
Yeah we can use None will update